### DISCUSSION TITLE
How do I connect to the authentication emulator?

### DISCUSSION THREAD

**Discussion by patelnets** [invertase/react-native-firebase#4510] [Q&A] [answered]
I would like to test my app's authentication flow locally. Firebase provides an [emulator for authentication](https://firebase.google.com/docs/emulator-suite/connect_auth#web) which seems like a really good option. But I can't find any documentation in this repo hinting at how to do this. I see there is documentation for connecting to the Firestore emulator but not the authentication emulator. I've read the firebase docs and didn't find them helpful. I've also had a look the source code in this library ( which is sweeeet btw!) but that hasn't helped me too much either, but I can't say I exactly understand all of it. 

Is there a way to do this? If not then can someone point me in an alternative direction to achieve this goal?

**Comment by patelnets** [comment, marked as answer]
Sorry! Never mind actually just saw this discussion ... https://github.com/invertase/react-native-firebase/discussions/4461

**Reply by mikehardy** [reply]
v10, just released, has auth emulator support! Enjoy
